### DISCUSSION TITLE
Thanks Jyers and the community

### DISCUSSION THREAD

**Discussion by HermanGaviriaMarin** [Jyers/Marlin#1187] [General Discussion]
I know we're usually here in the discussion page usually asking for help and also complaining about things not working. Many times we all forget to thank the creator(s), this is a really cool firmware and without all the little features and shortcuts it provides to avoid the need to do dozens of uncommenting in visual studio code and also reduce the need for pronterface. So for all the hard work and patience I'd like to say thank you for the firmware and all the tips.

**Comment by Cal-LF** [comment]
Jacob has made the E3V2 printer into a better machine. For me, his project has been a good learning experience. I think learning to compile features and changes into the firmware is keeping the mind active. Along with learning to use the simpler CAD design software to make my own things.
Supporting Mr. Myers is a great way to give our thanks. A few bucks to Pay Pal or the Patreon will help keep the project going.
